app/main.py

```diff
@@ -2,0 +1,23 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command_split = command.split()
+    source_file_name = command_split[1]
+    directory = "/".join(command_split[2].split("/")[:-1])
+    destination_file_name = command_split[2].split("/")[-1]
+
+    print(source_file_name)
+    print(directory)
```

Comment:
Remove debugging print statements to clean up the code before deploying or sharing it.